### PR TITLE
Added PTransform convenience methods for EE ingestion.

### PR DESCRIPTION
**Pull request by alxmrs**
I've enhanced the PTransform in our EE pipeline to parse key-word arguments, for example, from a dataclass or argparse. This will hopefully help reduce the boilerplate of passing arguments in `weather-mv ee` pipelines.

### CODE REVIEW

File: weather_mv/loader_pipeline/sinks.py [hunk 1]
```diff
@@ -42,18 +43,25 @@
 logger.setLevel(logging.INFO)
 
 
+class KwargsFactoryMixin:
```
Comment (deepgabani8) [inline weather_mv/loader_pipeline/sinks.py:46]: Wow! Thanks Alex.

File: weather_mv/loader_pipeline/ee.py [hunk 2]
```diff
@@ -403,7 +380,7 @@ def process(self, uri: str) -> t.Iterator[str]:
 
 
 @dataclasses.dataclass
-class ConvertToAsset(beam.DoFn):
+class ConvertToAsset(beam.DoFn, beam.PTransform, KwargsFactoryMixin):
```
Comment (deepgabani8) [inline weather_mv/loader_pipeline/ee.py:383]: Can we remove `beam.DoFn` now?

Comment (alxmrs) [reply weather_mv/loader_pipeline/ee.py:383]: In theory, we could. I'm keeping it for now for backwards compatibility.

File: weather_mv/loader_pipeline/sinks.py (hunk 1 above)
Comment (alxmrs) [reply weather_mv/loader_pipeline/sinks.py:46]: :)

**Review by deepgabani8 — APPROVED** [review]

**Pull request merged**